project.clj

```diff
@@ -48,22 +55,19 @@
   ;; code that we have.
   :classifiers [["test" :testutils]]
 
-  :profiles {:cljfmt {:plugins [[lein-cljfmt "0.5.0"]
-                                [lein-parent "0.2.1"]]
+  :profiles {:cljfmt {:plugins [[lein-cljfmt "0.5.0"]]
                       :parent-project {:path "ext/pl-clojure-style/project.clj"
                                        :inherit [:cljfmt]}}
              :dev {:source-paths ["examples/shutdown_app/src"
                                   "examples/java_service/src/clj"]
                    :java-source-paths ["examples/java_service/src/java"]
-                   :dependencies [[spyscope "0.1.4"]
-                                  [puppetlabs/kitchensink ~ks-version :classifier "test"]]
-                   :injections [(require 'spyscope.core)]}
+                   :dependencies [[puppetlabs/kitchensink nil :classifier "test"]]}
```

Comment:
So `nil` here will be substituted with the inherited `managed-dependencies` reference to `kitchensink` from `clj-parent`?